.travis.yml

```diff
@@ -1,5 +1,16 @@
 language: r
+r: 
+  - 3.4.2
 sudo: required
+before_install:
```

Comment:
Not sure whats going on here? Lots of changes in travis file. We should've only needed 3.4.2 right?

Reply:
To be fair, I didn't know the extent of what we needed, so I just added everything! I'll take the extraneous stuff out.